ember-toucan-core/rollup.config.mjs

```diff
@@ -24,7 +24,11 @@ export default {
     // These are the modules that should get reexported into the traditional
     // "app" tree. Things in here should also be in publicEntrypoints above, but
     // not everything in publicEntrypoints necessarily needs to go here.
-    addon.appReexports(['components/*.js', 'components/form/*.js']),
+    addon.appReexports([
+      'components/*.js',
+      'components/form/*.js',
+      'components/form/controls/*.js',
+    ]),
```

Comment:
If you rebase, you won't need to adjust the rollup config any longer!

(For ref: https://github.com/CrowdStrike/ember-toucan-core/pull/36/files#diff-ef1c3fa189e4cf0ff2a244e5f3ad25086f8d12247f8a37c66e2dc1a8aa8cebb7)